Controller/LogController.php

```diff
@@ -20,7 +20,7 @@ public function createAction(Request $request)
         if(Request::METHOD_GET === $request->getMethod()){
             $level = (string) $request->query->get('level');
             $message = (string) $request->query->get('msg');
-            $context = (array) $request->query->get('context', array());
+            $context = (array) $request->query->all('context', array());
```

Comment:
the code should be simplified:

- `all` does not have a second argument (the default value is always an array
- the array cast is useless